rules/cross-platform/defense_evasion_masquerading_space_after_filename.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2022/10/18"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/10/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rules identifies a process created from an executable with a space appended to the end of the filename. This may
+indicate an attempt to masquerade a malicious file as benign to gain user execution. When a space is added to the end of
+certain files, the OS will execute the file according to it's true filetype instead of it's extension. Adversaries can
+hide a program's true filetype by changing the extension of the file. They can then add a space to the end of the name so
+that the OS automatically executes the file when it's double-clicked.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Masquerading Space After Filename"
+note = """## Setup
+
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+"""
+references = [
+    "https://www.picussecurity.com/resource/blog/picus-10-critical-mitre-attck-techniques-t1036-masquerading",
+]
+risk_score = 47
+rule_id = "f5fb4598-4f10-11ed-bdc3-0242ac120002"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "macOS", "Threat Detection", "Defense Evasion"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and 
```

Comment:
Since this is more for Linux or macOS, I would add that as a filter with something like `host.os.type : ("linux","macos")`. The reason being `process.name` and `process.args` are too generic to filter out the potential for Windows triggering this alert.

Reply:
Good point, thank you I will update the query